Tests/FluentAssertions.Specs/Collections/CollectionAssertionSpecs.BeProperSubsetOf.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using FluentAssertions.Execution;
+using Xunit;
+using Xunit.Sdk;
+
+namespace FluentAssertions.Specs.Collections;
+
+/// <content>
+/// The BeProperSubsetOf specs.
+/// </content>
+public partial class CollectionAssertionSpecs
+{
+    public class BeProperSubsetOf
+    {
+        [Fact]
+        public void When_collection_is_proper_subset_of_a_specified_collection_it_should_not_throw()
```

Comment:
🔧 We prefer to avoid using terms like _when_ and _should_.

Reply:
I see many assertions use this styling (ex: `CollectionAssertionSpecs.BeNullOrEmpty`), 
what would be a better description for this?

Reply:
That is because we started changing the style a year or two ago. We now prefer a more factual style. 

In this case, `A_collection_with_only_items_of_a_superset_but_misses_some_is_a_proper_subset`. It'll take a bit more time to come up with but has really helped us and contributors understand the subtleties of our test cases.